src/App.tsx

```diff
@@ -1,109 +1,75 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { ItemPerPage } from './types/ItemPerPage';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
u can always get only the items u need. Do it instead `line 22`
```suggestion
const items = getNumbers(startIndex, endIndex).map(n => `Item ${n}`);
```